toolset/run-tests.py

```diff
@@ -208,6 +208,10 @@ def main(argv=None):
         nargs='*',
         default=None,
         help='Extra docker arguments to be passed to the test container')
+    parser.add_argument(
+        '--force-rm',
+        default=True,
+        help='Remove intermediate docker containers after running.')
```

Comment:
I think that if we're going to do this in the aim of improving ergonomics for contributors who need to quickly iterate implementations again and again and want those layers to persist between attempts, then this should be `False` by default, and we need to update our CI scripts to forcibly set to `True` (and our continuous scripts).